src/engine/Drawing/Polygon.ts

```diff
@@ -71,21 +71,21 @@ export class Polygon implements IDrawable {
    /**
     * @notimplemented Effects are not supported on `Polygon`
     */
-   public addEffect(effect: Effects.ISpriteEffect) {
+   public addEffect() {
```

Comment:
We can't remove the methods due to the interface requiring them. That could be addressed in #652